crates/wasm-wave/src/parser.rs

```diff
@@ -176,7 +182,7 @@ impl<'source> Parser<'source> {
     }
 
     fn finish_flags(&mut self, start: usize) -> Result<Node, ParserError> {
-        let mut flags = IndexMap::with_capacity(1);
+        let mut flags = BTreeMap::new();
```

Comment:
This is a (small) functional change, switching the ordering of serialized flags from definition-ordered to lexicographical. I don't see this behavior documented anywhere so its probably fine. Clearly it wasn't important enough to me to assert it in a test case. 😅